server.xml

```diff
@@ -19,7 +19,7 @@
      define subcomponents such as "Valves" at this level.
      Documentation at /docs/config/server.html
  -->
-<Server port="8005" shutdown="SHUTDOWN">
+<Server port="-1">
```

Comment:
Does `-1` have a special meaning for Tomcat?

Reply:
yeah, means shutdown port is disabled.